### ISSUE TITLE
unable to open file even after converting to .zip format

### ISSUE THREAD

**Issue by merlinbry** [olivierkes/manuskript#776]
Hi,
I was writing my thesis  on this and it shut down automatically.  

![Screenshot 2020-06-03 at 4 57 24 PM](https://user-images.githubusercontent.com/62759183/83652778-4dbe3580-a5bb-11ea-89c5-827f1eeb4077.png)

It won't open the file and it showed fatal error.And as I tried to convert back to  .msk format to post the error here, manuskript doesn't even detect the file.

**Comment by merlinbry** [comment]
@olivierkes could you please lookinto it.I didn't have a back up and all the work until now is in that file.

**Comment by merlinbry** [comment]
@gedakc

**Comment by merlinbry** [comment]
![Screenshot 2020-06-04 at 10 45 11 AM](https://user-images.githubusercontent.com/62759183/83735289-793f2f80-a650-11ea-84e0-bebdd48e8e03.png)

**Comment by gedakc** [comment]
Is the .msk file zero bytes?  If so then there is no data in it to be recovered.

**Comment by merlinbry** [comment]
@gedakc yes, I again changed  .zip to .msk inorder to verify and it shows zero byte.I was working with manuscript and then it crashed.Afterwards I tried the #319 ,changed it to zip.It showed zero bytes.

**Comment by gedakc** [comment]
Did you make a backup of your .msk before you changed the name to .zip?

**Comment by merlinbry** [comment]
@gedakc No...

**Comment by gedakc** [comment]
If you have no other backup, and the file you have is zero bytes, then the data is lost.  ;-(

**Comment by gedakc** [comment]
The first step when trying to recover any type of data is to make a backup of what you currently have so that if something goes wrong then at least you can restore back to that state.

My guess (and this is only a guess) is that the zip file was corrupt and the software used to open the zip file opened it as writable, encountered an error, aborted by re-writing out zero bytes and closed the file.  This is only a guess.

**Comment by merlinbry** [comment]
@gedakc  I didn't.I checked the folder before looking for anything, it wasn't there. I have a backup copy from February.

**Comment by ptoche** [comment]
make a thorough disk search, using an app that allows you to search invisible files, search for any msk/zip files and for any file modified around the time of the tragedy, mayb a backup exists somewhere. Could be the zip app makes a backup, who knows. Miracles happen. Good luck.
